tests/e2e/Services/Projects/ProjectsDevelopmentKeys.php

```diff
@@ -0,0 +1,258 @@
+<?php
+
+namespace Tests\E2E\Services\Projects;
+
+use Tests\E2E\Client;
+use Utopia\Database\DateTime;
+
+trait ProjectsDevelopmentKeys
+{
+    /**
+     * @depends testCreateProject
+     * @group developmentKeys
+     */
+    public function testCreateProjectDevelopmentKey($data): array
+    {
+        $id = $data['projectId'] ?? '';
+
+        $response = $this->client->call(Client::METHOD_POST, '/projects/' . $id . '/development-keys', array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+        ], $this->getHeaders()), [
+            'name' => 'Key Test',
+            'expire' => DateTime::addSeconds(new \DateTime(), 36000)
```

Comment:
Let's expire in 10 seconds. After tests have sleep for 10 seconds, and try to use it to get failure.